### PR TITLE
W-15246380 fhir asset desc

### PR DESCRIPTION
**Pull request by tasmoore**

### CODE REVIEW

File: healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
-* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/fhir-error-policy/[FHIR Error API Policy] - Captures error responses and formats them in an FHIR OperationOutcome-compliant format, since Anypoint Platform's out-of-the-box (OOTB) policies do not provide error messages in an FHIR-compliant format. This policy can also be used with other OOTB API policies provided in Anypoint Platform, such as OAuth or JWT.
-* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/hc-accelerator-okta-smartfhir/[Okta SMART on FHIR^] - Implementation template that walks you through how to set up a reference SMART on FHIR implementation on Anypoint Platform using Okta as the identity provider.
+* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/hls-fhirjson-to-raml-sys-api-spec/[FHIR RAML auto-generator] - Utility to generate RAML specifications for FHIR resources from the JSON schema provided by HL7.
+* xref:fhir-r4-us-core-profiles.adoc[FHIR R4 US Core Profile Assets] - A list of API specifications, data types, and libraries to meet FHIR R4 USCDI Core Profile compliance standards.
```
Comment (rstastny9) [inline healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc]: Can we change "list" to "group"? Want to make sure the user knows this is a collection of assets not just a list of items. We could use collection or group - whatever you think is best

Comment (rstastny9) [reply healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc]: Let's remove R4 from this sentence

File: healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
+* xref:fhir-r4-resources.adoc[FHIR R4 Resources List] - A list of API specifications, data types, and libraries to meet FHIR R4 compliance standards, as well as information about the FHIR Error API policy.
```
Comment (rstastny9) [inline healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc]: Same feedback as above re: list. Also let's remove reference of the Error Policy since that is the next asset we provide

File: healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc [hunk 3]
```diff
@@ -2,11 +2,11 @@
...
+* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/fhir-error-policy/[FHIR Error API Policy] - A guide for capturing error responses and formatting them in FHIR OperationOutcome-compliant format.
```
Comment (rstastny9) [inline healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc]: "A custom policy that captures error responses and formats them into a FHIR compliant format (OperationOutcome)."

**Review by rstastny9 — CHANGES_REQUESTED** [review]
Praveen and I reviewed this together and provided our feedback. Thanks so much for pushing this forward!

File: healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc (hunk 1 above)
Comment (tasmoore) [reply healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc]: "collection" sounds good

File: healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc (hunk 3 above)
Comment (tasmoore) [reply healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc]: We try to avoid using parentheses, so I'm rewording to remove them.

File: healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc [hunk 4]
```diff
@@ -2,11 +2,11 @@
...
-* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/hc-accelerator-okta-smartfhir/[Okta SMART on FHIR^] - Implementation template that walks you through how to set up a reference SMART on FHIR implementation on Anypoint Platform using Okta as the identity provider.
+* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/hls-fhirjson-to-raml-sys-api-spec/[FHIR RAML auto-generator] - Utility to generate RAML specifications for FHIR resources from the JSON schema provided by HL7.
+* xref:fhir-r4-us-core-profiles.adoc[FHIR R4 US Core Profile Assets] - A collection of API specifications, data types, and libraries designed to meet FHIR USCDI Core Profile compliance standards.
+* xref:fhir-r4-resources.adoc[FHIR R4 Resources List] - A collection of API specifications, data types, and libraries designed to meet FHIR R4 compliance standards.
```
Comment (rstastny9) [inline healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc]: Can we rephrase this to "A collection of API specifications, data types, and libraries designed to align to the FHIR R4 base standard."?

File: healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc [hunk 5]
```diff
@@ -2,11 +2,11 @@
...
+* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/fhir-error-policy/[FHIR Error API Policy] - A custom policy that captures error responses and formats them into `OperationOutcome`, which is a FHIR compliant format.
```
Comment (rstastny9) [inline healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc]: @ppalu what do you think of this phrasing? Seems a little off to me but you would know better if this accurately captures what we had built

**Review by rstastny9 — CHANGES_REQUESTED** [review]
Left a couple final tweaks

File: healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc (hunk 4 above)
Comment (tasmoore) [reply healthcare/2.22/modules/ROOT/pages/fhir-r4-assets.adoc]: I updated this description and added the second sentence back in for more context:

"A custom policy that captures error responses and transforms them into an FHIR compliant `OperationOutcome` format. This policy can also be used with other out-of-the-box API policies provided in Anypoint Platform, such as OAuth or JWT."

**Review by rstastny9 — APPROVED** [review]
Looks good to me!
